httpsender/add-more-headers.js

```diff
@@ -0,0 +1,70 @@
+// This HttpSender script adds headers to all messages transmitted by zaproxy,
+// including automated tools. Refer to the HttpSender class definition:
+// https://github.com/zaproxy/zaproxy/blob/master/zap/src/main/java/org/parosproxy/paros/network/HttpSender.java
```

Comment:
Needs update.

Reply:
Done.

Reply:
Thanks, just circled back and you're ahead of me :)

Reply:
No problem, rushing for release :)